CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+
+# Contributing to `{project_name}`
```

Comment:
will this actually work? like, will github dynamically insert this in??

Reply:
Not really - I've replaced that placeholder with `the project`